cultural-fund-map/data/inventory.json

```diff
@@ -12,7 +12,7 @@
             "address": "100 W Oxford St",
             "zip": "19122",
             "website": "http://www.theartwell.org/",
-            "grantAmount": "50000",
+            "grantAmount": "11150",
```

Comment:
If you had `grantAmount` and `orgBudget` saved as numbers (ie, without the quotes), you would have more display formatting options.  For instance, you could use [toLocaleString](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString) to get comma's in there:

```javascript
(12345).toLocaleString('en') -> "12,345"
```